Feature

tkarsakov opened the pull request.

tkarsakov committed:
Add basic classes

tkarsakov committed:
Finalize functionality

Add missing text operations. Add user input method. Create Menu class.
Finalize UI in Menu class.

tkarsakov committed:
Correct wrong error description in searchWord method

MichaelRzheutski commented on .gitignore:
В .gitignore можно добавить папки: 
out/
target/
.idea/

поскольку они нам не нужны на гитхабе

MichaelRzheutski commented on pom.xml:
В помник можно добавить проперти типа:
```
    <properties>
        <log4jVersion>2.21.1</log4jVersion>
    </properties>
```
чтобы удобнее рулить версиями зависимостей

MichaelRzheutski commented on src/main/java/com/solvd/ui/Menu.java:
```diff
@@ -0,0 +1,69 @@
+    }
+
+    public void newText() {
+        LOGGER.info("Input raw text or a valid filename to load text from: ");
```
При запуске проги чуток непонятно откуда взять файл для чтения, может добавить указание на жёсткий диск? Например, **Input raw text or a valid path to file from hard drive to load text from:**

tkarsakov committed:
Add support for inputs in Russian. Clarify text loading process.

Modify regex in TextManager methods to correctly process inputs in Russian.
Add additional clarification for .newText method specifying file location, show
loaded text on screen after loading is complete.
Modify pom.xml to include properties for dependency version and such.
Add missing folders to .gitignore
